environment/learnrails/spec/controllers/articles_controller_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ArticlesControllerController, type: :controller do
```

Comment:
this is weird why name of class is ArticlesControllerController, 
why not just ArticlesController